llama_stack/cli/stack/list_stacks.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import argparse
+from pathlib import Path
+from typing import Dict
+from llama_stack.cli.subcommand import Subcommand
+from llama_stack.cli.table import print_table
+
+
+class StackListBuilds(Subcommand):
+     """List built stacks in .llama/distributions directory"""
+
+     def __init__(self, subparsers: argparse._SubParsersAction):
+        super().__init__()
+        self.parser = subparsers.add_parser(
+            "list",
+            prog="llama stack list",
+            description="list the build stacks",
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        )
+        self._add_arguments()
+        self.parser.set_defaults(func=self._list_stack_command)
+        
+     def _add_arguments(self):
+        self.parser.add_argument(
+            "--verbose",
+            "-v",
+            action="store_true",
+            help="Show additional details about each stack",
+        )
+
+     def _get_distribution_dirs(self) -> Dict[str, Path]:
+        """Return a dictionary of distribution names and their paths"""
+        distributions = {}
+        dist_dir = Path.home() / ".llama" / "distributions"
```

Comment:
This is fine for now, but we really should transition to XDG with platformdirs at some point.